src/content/platform-integration/android/call-jetpack-apis.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Calling JetPack APIs"
+description: "Utilize the latest Android APIs from your Dart code"
+---
+
+<?code-excerpt path-base="platform_integration"?>
+
+Flutter apps running on Android can always make use of the
+latest APIs, the first day they are released on Android, no
+matter what. This page outlines available ways to invoke
+Android-specific APIs.
+
+## Use an existing solution
+
+In most scenarios, you can [use a plugin][use-a-plugin] to
+invoke native APIs without writing any custom boilerplate or
+glue code yourself.
+
+[use-a-plugin]: /platform-integration/android/call-jetpack-apis#use-a-plugin
+
+### Use a plugin
+
+Using a plugin is often the easiest way to access native
+APIs, regardless of where your Flutter app is running. To
+use plugins, visit [pub.dev][pub] and search for
+the topic you need. Most native features, including accessing
```

Comment:
Should we filter to the Android platform, ie by searching [`platform:android`](https://pub.dev/packages?q=platform%3Aandroid)? Just a thought, I don't feel strongly

Reply:
Personally, even though Android requested this documentation, my perspective was still to write it such that it works for everyone.

Thoughts?

Reply:
Yup sounds good